scripts/release.js

```diff
@@ -20,29 +22,57 @@ const packageJson = require('../package.json');
 
 const {showChangelog, getChangelog, updateChangelog} = require('./lib/conventionalChangelog.js');
 
-shell.echo(`Algoliasearch-Helper release script`);
+const {canary: isCanary} = argv;
+
+if (isCanary) {
+  releaseCanaryVersion();
+} else {
+  releaseStableVersion();
+}
 
-checkEnvironment();
-mergeDevIntoMaster();
-showChangelog(shell);
-promptVersion(packageJson.version, (version) => {
-  bumpVersion(version, () => {
+function releaseCanaryVersion() {
+  shell.echo(`Algoliasearch-Helper release CANARY version`);
```

Comment:
```suggestion
  shell.echo(`AlgoliaSearch Helper release CANARY version`);
```

Reply:
the package name is `algoliasearch-helper` though

Reply:
We should either go with the package name ("algoliasearch-helper") or the library title ("AlgoliaSearch Helper").

Reply:
Yep, will use the name that we have on the Community website which is: Algolia JS Helper.

Reply:
https://github.com/algolia/algoliasearch-helper-js/pull/712/commits/6ebe9c7a44f33183c3cf04afc3285369663cebb6